install/kubernetes/cilium/values.yaml

```diff
@@ -1339,3 +1342,124 @@ enableCriticalPriorityClass: true
 # disableEnvoyVersionCheck removes the check for Envoy, which can be useful on
 # AArch64 as the images do not currently ship a version of Envoy.
 #disableEnvoyVersionCheck: false
+
+# clustermesh-apiserver is created if externalWorkloads are enabled or if cluster.name != "default".
+clustermesh:
+  apiserver:
+    image:
+      repository: docker.io/cilium/clustermesh-apiserver
+      tag: latest
+      pullPolicy: IfNotPresent
+
+    etcd:
+      image:
+        repository: quay.io/coreos/etcd
+        tag: v3.4.13
+        pullPolicy: IfNotPresent
+
+    service:
+      type: NodePort
+      # optional port to use as the node port
+      nodePort: 32379
+      # optional loadBalancer IP to use with type LoadBalancer
+      # loadBalancerIP:
+
+      # For GKE LoadBalancer, use annotation cloud.google.com/load-balancer-type: "Internal"
+      # For EKS LoadBalancer, use annotation service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
+      annotations:
+      - test-annotation: "Foo"
```

Comment:
This is probably not supposed to be included?

Reply:
Right, will remove!

Reply:
Fixed!